src/applications/apps.js

```diff
@@ -40,7 +40,7 @@ export function getAppChanges() {
 
     switch (app.status) {
       case LOAD_ERROR:
-        if (currentTime - app.loadErrorTime >= 200) {
+        if (appShouldBeActive && currentTime - app.loadErrorTime >= 200) {
```

Comment:
Previously we would attempt to reload any application in LOAD_ERROR status, even if it shouldn't be active.